test/barcodes/test_barcodes.py

```diff
@@ -1,6 +1,10 @@
+from pathlib import Path
+
 from fpdf import FPDF
 from test.utilities import assert_pdf_equal
 
+HERE = Path(__file__).resolve().parent
```

Comment:
`PARENT_DIR` could be more explicit than just `HERE`.

This remarks applies to all other files

Reply:
Yes, naming is difficult and I was quite happy with `HERE`. I understand `PARENT_DIR` as the directory above other directory, which is not what we need here. I'd leave it as `HERE`, but if you wish, feel free to add another commit with renaming.